src/vendor/core/warning.js

```diff
@@ -40,7 +40,9 @@ if (__DEV__) {
 
     if (!condition) {
       var argIndex = 0;
-      console.warn('Warning: ' + format.replace(/%s/g, () => args[argIndex++]));
+      if (console != null) {
```

Comment:
This should be `typeof console !== 'undefined'` to avoid a `use strict` violation in the case that it is not defined on the global at all.